source/features/restore-file.tsx

```diff
@@ -48,17 +42,17 @@ async function getFile(filePath: string): Promise<{isTruncated: boolean; text: s
 	return repository.file;
 }
 
-async function restoreFile(progress: (message: string) => void, menuItem: Element, filePath: string): Promise<void> {
+async function dropFile(progress: (message: string) => void, menuItem: Element, filePath: string): Promise<void> {
 	const file = await getFile(filePath);
 
 	if (!file) {
 		// The file was created by this PR.
 		// This code won’t be reached if `highlight-deleted-and-added-files-in-diffs` works.
-		throw new Error('Nothing to restore. Delete file instead');
+		throw new Error('Nothing to drop. Delete file instead');
```

Comment:
That error message seems wrong now: There is something to drop (the whole new file), but nothing to restore. When does this error occur?

Suggestions for better wording:

* File was created in this PR. Delete it instead.
* Can't drop created file. Delete it instead.